iroha_client/benches/torii.rs

```diff
@@ -179,6 +185,81 @@ fn instruction_submits(criterion: &mut Criterion) {
     }
 }
 
-criterion_group!(instructions, instruction_submits);
+fn instruction_submits_network(criterion: &mut Criterion) {
+    let mut group = criterion.benchmark_group("instruction-requests-network");
+    let rt = Runtime::test();
+    let txs_in_block = 50;
+    let mut iroha_client = rt.block_on(setup_bench_network(4, txs_in_block));
+
+    let account_id = AccountId::new(ACCOUNT_NAME, DOMAIN_NAME);
+    let asset_definition_id = AssetDefinitionId::new("xor", DOMAIN_NAME);
+
+    let mut success_count = 0;
+    let mut failures_count = 0;
+
+    let _ = group.bench_function("instructions", |b| {
+        b.iter(|| {
+            let ev = iroha_client
+                .listen_for_events(EventFilter::Pipeline(pipeline::EventFilter {
+                    entity: None,
+                    hash: None,
+                }))
+                .unwrap();
+
+            let quantity = txs_in_block / 100;
+            for _ in 0..quantity {
+                let mint_asset = MintBox::new(
+                    Value::U32(1),
+                    IdBox::AssetId(AssetId::new(
+                        asset_definition_id.clone(),
+                        account_id.clone(),
+                    )),
+                );
+                if let Err(e) = iroha_client.submit(mint_asset) {
+                    eprintln!("Failed to execute instruction: {}", e);
+                }
+            }
+
+            let n: i32 = ev
+                .take(quantity as usize)
+                .map(|ev| {
+                    if matches!(
+                        ev,
+                        Ok(Event::Pipeline(pipeline::Event {
+                            status: pipeline::Status::Committed,
+                            ..
+                        }))
+                    ) {
+                        1
+                    } else {
+                        0
+                    }
+                })
+                .sum();
+
+            success_count += n;
+            failures_count += 100 - n;
+        })
+    });
+
+    println!(
+        "Success count: {}, Failures count: {}",
+        success_count, failures_count
+    );
+    group.finish();
+
```

Comment:
As our main aim is to calculate TPS then let's calculate and output it?

Reply:
And see the error count

Reply:
Like stress testing